README.md

```diff
@@ -0,0 +1,41 @@
+# katello-pull-transport-migrate
+
+_katello-pull-transport-migrate_ is a script which can be run on a client to migrate from katello-agent to REX pull transport (formerly known as "pull provider.")
+
+## What it does
+* Gets configuration from `subscription-manager config --list`
+* Writes correct configuration in config.toml based on subscription-manager config
+* Enables pull transport daemon service (yggdrasild) on the client and displays status
+
+After a successful run, it is safe to remove katello-agent from the client.
+
+## Prerequisites
+
+### Server
+(the following soon to be done for you by foreman-installer)
```

Comment:
Please open an issue for this (https://projects.theforeman.org/projects/puppet-foreman/issues/new) and link to it here.

Reply:
Opened https://projects.theforeman.org/issues/34239